src/sections/SMI/smi.style.js

```diff
@@ -0,0 +1,146 @@
+import styled from "styled-components";
+
+const SMIWrapper = styled.section`
+    .description{
+        margin: 6.25rem -0.94rem 7.5rem -0.94rem;
+        h6{
+            font-size: 20px;
+            color: #1E2117;
```

Comment:
Add this color to global them if not already present

Reply:
I think this is the jungle green color and it's already there in the theme.